experiments/remote_execute_api.yaml

```diff
@@ -0,0 +1,16 @@
+experiments:
+  - metadata:
+      name: remote-execute-api
+      type: remote_execute_api
+      namespace: default
+    parameters:
+      image: alconen/egress_server
+      # image_parameters are passed as individual environment variables, separated by semicolons
+      image_parameters: "URLS=https://google.com,https://linkedin.com,https://openai.com;EXAMPLE=example"
+      target_port: 4000
+      path: /experiment/CheckEgress/
+      service_account_name: default
+      local_port: 9500
+      expected_response: 200
+      expected_body: '{"Name":"CheckEgress","Success":true}'     
```

Comment:
I think it might be the case that users have some URLs blocked and expect others to work. I think we could customize the expected response to match their expectation per URL like google cloud is pass, but openai is fail. This is again to support users who want to use the image as is and just want to test different URLs.